File: web/src/api/campusUser.js
import service from '@/utils/request'

export const getCampusUserList = (params) => {
  return service({
    url: '/campusUser/getCampusUserList',
    method: 'get',
    params
  })
}

export const findCampusUser = (params) => {
  return service({
    url: '/campusUser/findCampusUser',
    method: 'get',
    params
  })
}

export const updateCampusUserStatus = (data) => {
  return service({
    url: '/campusUser/updateCampusUserStatus',
    method: 'post',
    data
  })
}

File: server/router/campus/campus_user.go
package campus

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type CampusUserRouter struct{}

func (r *CampusUserRouter) InitCampusUserRouter(Router *gin.RouterGroup) {
	userRouter := Router.Group("campusUser").Use(middleware.OperationRecord())
	userRouterWithoutRecord := Router.Group("campusUser")
	{
		userRouter.POST("updateCampusUserStatus", campusUserApi.UpdateCampusUserStatus)
	}
	{
		userRouterWithoutRecord.GET("getCampusUserList", campusUserApi.GetCampusUserList)
		userRouterWithoutRecord.GET("findCampusUser", campusUserApi.FindCampusUser)
	}
}
